src/main/paradox/docs/releases/v1.3-release-notes.md

```diff
@@ -0,0 +1,98 @@
+
+A CompositeView can define several sources of data of different types:
+
+* ProjectEventStream: the event log of the project where the CompositeView is defined
+* CrossProjectEventStream: the event log of an arbitrary project within the same Nexus deployment
+* RemoteProjectEventStream: the event log of an arbitrary project within a different Nexus deployment
+
+The sources are consumed as a collection of event streams, either from the primary store or via Server Sent Events in
+the case of the RemoteProjectEventStream. Each event is either exchanged with the resource for which it was emitted
+or dropped (if the filters defined in the originating source exclude it). The resource is then transformed into an
+RDF graph and stored in a temporary namespace of BlazeGraph.
+
+Projections are then executed by querying the temporary namespace via the `SPARQL CONSTRUCT` queries defined as part of
+their configuration. These queries allow constructing an arbitrary RDF graph by traversing the temporary one starting
+from the IRI node representing the originating resource id (the system recognizes the `{resource_id}` token in the
+provided query and replaces it with the corresponding value).
+
+CompositeViews allow defining multiple projections; the current supported types are:
+
+* ElasticSearchProjection: data is projected as documents in an ElasticSearch index
+* SparqlProjection: data is projected as an RDF named graph in a BlazeGraph namespace
+
+Projections include filtering configuration to allow resources of different types or constrained by different schemas to
+be indexed separately. An obvious use case is to index resources of different shapes in different ElasticSearch indices
+with appropriate mapping definitions for each variation.
+
+The complete configuration options can be found in the @ref:[api reference](../api/current/kg/views/composite-view-api.md).
+
+## Studio
+
+## View offsets
+
+Views now expose `offset` as a sub-resource allowing the retrieval of the last consumed event id. The event id values
+are the same as the ones presented via @ref:[Server Sent Events](../api/current/kg/kg-resources-api.md#resources-server-sent-events).
+
+```
+GET /v1/views/{org_label}/{project_label}/{view_id}/offset
+```
+
+An example response:
+
+```json
+{
+    "@context": "https://bluebrain.github.io/nexus/contexts/offset.json",
+    "@type": "TimeBasedOffset",
+    "instant": "2020-02-14T09:31:41.400Z",
+    "value": "ceaf0580-4f0c-11ea-98d9-dff6b2cf55b4"
+}
+```
+
+Deleting the `offset` sub-resource instructs the system to rebuild the corresponding index by restarting the event
+replay and indexing process.
+
+```
+DELETE /v1/views/{org_label}/{project_label}/{view_id}/offset
+```
+
+The system acknowledges the command by providing a `NoOffset` response:
+
+```json
+{
+    "@context": "https://bluebrain.github.io/nexus/contexts/offset.json",
+    "@type": "NoOffset"
+}
+```
+
```

Comment:
There should be a mention to the endpoints

- GET `/v1/views/{org_label}/{project_label}/{view_id}/projections/_/offset`
Fetch the projections offset

- DELETE `/v1/views/{org_label}/{project_label}/{view_id}/projections/_/offset`
rebuild the projections, without rebuilding the temporary blazegraph query space. This allows for rebuilding the projections with potentially greater amount of triples in the temporary blazegraph space

Also, required mention of additions & improvements on listings API:
- Added `sort` query param.
- Improved the results from `q` query param.

Reply:
Done.